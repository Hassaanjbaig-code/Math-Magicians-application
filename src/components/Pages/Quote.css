@import url('https://fonts.googleapis.com/css2?family=Noto+Serif:ital,wght@0,400;0,700;1,400;1,700&display=swap');

.Quote-main {
  width: 100%;
  display: flex;
  align-items: center;
  justify-content: center;
  height: 70vh;
}

.Quote-main p {
  font-family: 'Noto Serif', serif;
  font-weight: 500;
  font-size: 1.5rem;
  line-height: 36px;
}
