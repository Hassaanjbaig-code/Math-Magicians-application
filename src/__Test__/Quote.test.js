import React from 'react';
import TestRenderer from 'react-test-renderer';
import Quote from '../components/Pages/Quote';

test('should snapshot', () => {
  const tree = TestRenderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
