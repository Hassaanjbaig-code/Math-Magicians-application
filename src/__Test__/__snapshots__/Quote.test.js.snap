// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should snapshot 1`] = `
<div
  className="Quote-main"
>
  <p>
    Mathematics is not about numbers, equations. computations, or algorithms: it is about understanding. â€”William Paul Thurston
  </p>
</div>
`;
